.erb/configs/webpack.config.renderer.dev.ts

```diff
@@ -170,7 +170,6 @@ export default merge(baseConfig, {
     },
     historyApiFallback: {
       verbose: true,
-      disableDotRule: false,
```

Comment:
Why change this?

Reply:
The typings state that it should be true or undefined so we should just remove it.

Typings: [typings](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b70b8e239fc881b20346ca659f60cfa5b08d980a/types/connect-history-api-fallback/index.d.ts#L19)

Reply:
Yes, @UserJHansen is right. That's why I remove it.